tests/integration/test_cloudformation.py

```diff
@@ -543,13 +575,15 @@ def test_create_cfn_lambda_without_function_name(self):
         lambdas_before = len(rs['Functions'])
 
         stack_name = 'stack-%s' % short_uid()
+        lambda_role_name = 'lambda-role-%s' % short_uid()
+        TEST_TEMPLATE_7['Resources']['LambdaExecutionRole']['Properties']['RoleName'] = lambda_role_name
```

Comment:
nit: instead of modifying the global variable directly (which is assumed to be a constant, with no side-effects), we could use `clone(..)` from `utils/common.py` and modify a local copy:
```
template = clone(TEST_TEMPLATE_7)
template['Resources']['LambdaExecutionRole']['Properties']['RoleName'] = lambda_role_name
...
```

Reply:
@duongpv7 Going to merge the PR anyway, but something to keep in mind for future iterations. Thanks